src/tailwind.js

```diff
@@ -8,6 +8,14 @@ const localConfig = {
     "./node_modules/@govtechsg/tradetrust-ui-components/src/**/*.{ts,tsx,js,jsx}",
   ],
   theme: {
+    screens: {
+      xs: "460px",
+      sm: "640px",
+      md: "768px",
+      lg: "1024px",
+      xl: "1280px",
+      "2xl": "1536px",
+    },
```

Comment:
https://tailwindcss.com/docs/screens#adding-smaller-breakpoints

Reply:
ok will update this accordingly.